doc/api/globals.markdown

```diff
@@ -2,188 +2,250 @@
+define what a module exports and makes available through `require()`.
 
-## require()
+See the [Module system documentation][] for more information.
+
+## Global and Module Scope
+
+Certain variables are defined at *both* the global and module scopes. The
+specific operation of these can vary depending on the current scope in
+which JavaScript code is executing.
+
+### `require()`
 
 <!-- type=var -->
 
 * {Function}
 
-To require modules. See the [Modules][] section.  `require` isn't actually a
-global but rather local to each module.
+The `require()` method is the primary mechanism for importing the functionality
+of another module into the current module. See the
+[Module system documentation][] for more information.
 
-### require.cache
+### `require.cache`
 
 * {Object}
 
 Modules are cached in this object when they are required. By deleting a key
-value from this object, the next `require` will reload the module.
+value from this object, the next `require()` will reload the module.
 
-### require.extensions
+### `require.extensions`
 
     Stability: 0 - Deprecated
 
 * {Object}
 
-Instruct `require` on how to handle certain file extensions.
-
-Process files with the extension `.sjs` as `.js`:
+Used to instruct `require` on how to handle certain file extensions.
 
-    require.extensions['.sjs'] = require.extensions['.js'];
-
-**Deprecated**  In the past, this list has been used to load
+**Deprecated**: Historically, this list has been used to load
 non-JavaScript modules into Node.js by compiling them on-demand.
-However, in practice, there are much better ways to do this, such as
+In practice, however, there are much better ways to do this, such as
 loading modules via some other Node.js program, or compiling them to
 JavaScript ahead of time.
 
-Since the Module system is locked, this feature will probably never go
-away.  However, it may have subtle bugs and complexities that are best
-left untouched.
-
-### require.resolve()
-
-Use the internal `require()` machinery to look up the location of a module,
-but rather than loading the module, just return the resolved filename.
+Because the Module system is Node.js is [Locked][], it is unlikely that this
+feature will be removed entirely. However, the mechanism is considered to be
+unsupported and should not be used.
 
-## setInterval(cb, ms)
+Note that files with the extension `*.sjs` are processed as `*.js` files.
 
-Run callback `cb` repeatedly every `ms` milliseconds. Note that the actual
-interval may vary, depending on external factors like OS timer granularity and
-system load. It's never less than `ms` but it may be longer.
-
-The interval must be in the range of 1-2,147,483,647 inclusive. If the value is
-outside that range, it's changed to 1 millisecond. Broadly speaking, a timer
-cannot span more than 24.8 days.
-
-Returns an opaque value that represents the timer.
-
-## setTimeout(cb, ms)
+    require.extensions['.sjs'] = require.extensions['.js'];
```

Comment:
This almost makes it sound like this is the case currently. Maybe we change the line above to 

"Note that below, files with the extension `*.sjs` are processed as `*.js` files." or something like that.